src/Microsoft.OpenApi.CSharpAnnotations.DocumentGeneration/PreprocessingOperationFilters/PopulateInAttributeFilter.cs

```diff
@@ -25,48 +27,68 @@ public class PopulateInAttributeFilter : IPreProcessingOperationFilter
         /// <param name="paths">The paths to be updated.</param>
         /// <param name="element">The xml element representing an operation in the annotation xml.</param>
         /// <param name="settings">The operation filter settings.</param>
-        public void Apply(OpenApiPaths paths, XElement element, PreProcessingOperationFilterSettings settings)
+        /// <returns>The list of generation errors, if any produced when processing the filter."></returns>
```

Comment:
">

Reply:
Sure will fix them all.

---
In reply to: [284025876](https://github.com/microsoft/OpenAPI.NET.CSharpAnnotations/pull/208#discussion_r284025876) [](ancestors = 284025876)